### PR TITLE
Fixed css navbar + advanced serviceForm

### PR DESCRIPTION
**Pull request by mariehessen**

### CODE REVIEW

File: src/components/containers/Header.js [hunk 1]
```diff
@@ -10,27 +11,19 @@ export default class Header extends Component {
...
-            <FlatButton>
-                <a href="/AdminPage">Services</a>
-            </FlatButton>,
-            <FlatButton>
-                Places
-            </FlatButton>,
-            <FlatButton>
-                <a href="/ServiceForm">New Service</a>
-            </FlatButton>
+            <FlatButton  hoverColor='transparent' rippleColor='transparent' labelStyle={{color: 'white'}} href="/AdminPage" label="Service" />,
```
Comment (Aborysa) [inline src/components/containers/Header.js]: FlatButton should not have a href field, but should be embeded in a react-router-dom Link element

File: src/components/containers/Header.js [hunk 2]
```diff
@@ -10,27 +11,19 @@ export default class Header extends Component {
...
         return(
             <AppBar
                 title="SIMS"
+                titleStyle={{marginLeft: '30%', fontSize: '220%' }}
                 iconElementLeft={
-                    <FlatButton>{buttons}</FlatButton>
-                }
-            >
-        
+                    <FlatButton hoverColor='transparent' rippleColor='transparent'>{buttons}</FlatButton>
```
Comment (Aborysa) [inline src/components/containers/Header.js]: FlatButton should not embed other FlatButtons

File: src/components/containers/ServiceForm/ServiceForm.js [hunk 3]
```diff
@@ -37,16 +51,20 @@ export default class ServiceForm extends Component {
...
 
-                    <h6>End date</h6>
-                    <DatePicker className='formtext' hintText="End date" /><br></br>
+
+                {/**********************************************************.*********************************************/}
+                {/********************************************** ADVANCED FIELD IN FORM **********************************/}
+                {/**********************************************************.*********************************************/}
+            
+
+                <div style={ showAdvanced }>
```
Comment (Aborysa) [inline src/components/containers/ServiceForm/ServiceForm.js]: It is better to not render the advanced block at all than to hide it using inline-styling

File: src/components/containers/ServiceForm/ServiceForm.js [hunk 4]
```diff
@@ -10,8 +10,15 @@ import MenuItem from 'material-ui/MenuItem';
 export default class ServiceForm extends Component {
     constructor(props){
         super(props);
-        this.state = {value: 1, verdi: 0};
+        this.state = {value: 1, verdi: 0, showAdvanced: false};
```
Comment (Aborysa) [inline src/components/containers/ServiceForm/ServiceForm.js]: having a `value` and a `verdi` field isn't very descriptive, and we should avoid mixing languages like that.

File: src/components/containers/Header.js (hunk 1 above)
Comment (mariehessen) [reply src/components/containers/Header.js]: image

**Pull request merged**